### ISSUE TITLE
Guards couldn't care less if citizens got attacked by a non-hostile mob

### ISSUE THREAD

**Issue by warning-djkajslh** [ldtteam/minecolonies#7722]
### Prerequisites 

- [X] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [X] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version:
- MineColonies Version: minecolonies-1.0.85-ALPHA-universal.jar
- Structurize Version:
- Related Mods and their Versions:

## Expected behavior

They would come rushing toward the mob that attack the citizen

## Actual behavior

They completely ignore the mob, not until it attack a guard that they will retaliate

## Steps to reproduce the issue

Tested with dragons and dread mobs from Ice and Fire.

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Same issue as the other one
